src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+@import './variables';
+@import './mixins';
+
+.stars {
+  display: flex;
+
+  &__star {
+    width: $star-size;
+    height: $star-size;
+    background-image: url('./images/star-empty.png');
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    &.stars--4 &__star:nth-child(-n + 4) {
+      background-image: url('./images/star-active.png');
+    }
```

Comment:
The selector `&.stars--4 &__star:nth-child(-n + 4)` is incorrect. It should be `.stars--4 &__star:nth-child(-n + 4)` to correctly apply the active star background image to the first four stars. The `&` is used incorrectly here, as it refers to the parent selector, which is `.stars__star` in this context. Adjust the selector to ensure it targets the correct elements.